src/instructlab/server.py

```diff
@@ -163,6 +164,19 @@ def server(
     queue=None,
 ):
     """Start OpenAI-compatible server"""
+
+    class ModelPathError(Exception):
+        pass
+
+    try:
+        if not Path(model_path).is_file():
+            raise ModelPathError(
+                f"Model file does not exist: '{model_path}'. If the model doesn't exist, download it using the 'ilab model download' command. Please verify the model path in 'config.yaml' or pass '--model-path'."
+            )
+    except ModelPathError as e:
+        logger.error(str(e))
+        raise e
```

Comment:
@leseb I acked your suggestion quickly, but I don't think I thought it through...

now the experience is a big traceback. I don't think that's what we want...

Reply:
Actually, this is a bit more complex than I thought. It might be easier to do the validation in the caller earlier and return a click error.

Reply:
Does the code have a central mechanism for sanitizing all user inputs, including defaults, YAML configurations, and parameters? Regardless of the command being executed, there should be a consistent quality check for all inputs, correct?

For instance, we could have a function like this:

```python
def sanitize_user_input(input):
  if input_is_of_model_path_type:
      perform_model_path_validation
  if input_is_of_model_family_type:
      perform_model_family_validation
```

This function would ensure that all user inputs undergo the necessary validation checks based on their type. Otherwise, I find it very error prone to introduce checks for the same thing, in several place. e.g. this exact check already exists in the code, in a different place 

see debug-llama:

```python
if not os.path.isfile(args.model):
        print(f"ERROR: Model file '{args.model}' is missing.")
        print("Run 'lab init' and 'lab download'.")
        sys.exit(2)
```

It's the same code, in different file.